docs/root/intro/arch_overview/upstream/aggregate_cluster.rst

```diff
@@ -154,42 +154,133 @@ The aggregate cluster uses a tiered load balancing algorithm with two main steps
    ``80%`` healthy, multiplying by ``1.4`` results in ``112%``, which is capped at ``100%``. In other words, any product
    above ``100%`` is treated as ``100%``.
 
-The example shows how the aggregate cluster level load balancer selects the cluster. E.g., healths
-of {{20, 20, 10}, {25, 25}} would result in a priority load of {{28%, 28%, 14%}, {30%, 0%}} of
-traffic. When normalized total health drops below 100, traffic is distributed after normalizing the
-levels' health scores to that sub-100 total. E.g. healths of {{20, 0, 0}, {20, 0}} (yielding a
-normalized total health of 56) would be normalized and each cluster will receive 20 * 1.4 / 56 = 50%
-of the traffic which results in a priority load of {{50%, 0%, 0%}, {50%, 0%, 0%}} of traffic.
-
-The load balancer reuses priority level logic to help with the cluster selection. The priority level
-logic works with integer health scores. The health score of a level is (percent of healthy hosts in
-the level) * (overprovisioning factor), capped at 100%. P=0 endpoints receive level 0's health
-score percent of the traffic, with the rest flowing to P=1 (assuming P=1 is 100% healthy - more on
-that later). The integer percents of traffic that each cluster receives are collectively called the
-system's "cluster priority load". For instance, for primary cluster, when 20% of P=0 endpoints are
-healthy, 20% of P=1 endpoints are healthy, and 10% of P=2 endpoints are healthy; for secondary, when
-25% of P=0 endpoints are healthy and 25% of P=1 endpoints are healthy. The primary cluster will
-receive 20% * 1.4 + 20% * 1.4 + 10% * 1.4 = 70% of the traffic. The secondary cluster will receive
-min(100 - 70, 25% * 1.4 + 25% * 1.4) = 30% of the traffic. The traffic to all clusters sum up to
-100. The normalized health score and priority load are pre-computed before selecting the cluster and
-priority.
+The aggregate cluster load balancer first calculates each priority's health score for every cluster, sums those up,
+and then assigns traffic based on the overall total. If the total is at least ``100``, the combined traffic is capped at
+``100%``. If it's below ``100``, Envoy scales (normalizes) it so that the final distribution sums to ``100%``.
```

Comment:
our rules are pretty shakey on this, but i think the percentages dont want to be literals - just config settings, response codes, etc - these are actual literals rather than just numbers

Reply:
Make sense. So should we leave them as it or should I make them bold using `**`?

Reply:
not sure would need to look at it rendered - generally i think boldening (which i like) is mostly useful for skim reading - so if it assists that

Reply:
@phlax I [updated](https://storage.googleapis.com/envoy-pr/305d892/docs/intro/arch_overview/upstream/aggregate_cluster.html) the style. Let me know if this looks better.